twitter_ads/http.py

```diff
@@ -257,7 +260,11 @@ def perform(self):
                     break
                 bytes_start = bytes_read
                 bytes_read += len(bytes)
-                self.__upload_chunk(location, chunk_size, bytes, bytes_start, bytes_read)
+                response = self.__upload_chunk(location, chunk_size, bytes, bytes_start, bytes_read)
+                response_time = int(response.headers['x-response-time'])
+                chunk_size = min_chunk_size * size(self._DEFAULT_CHUNK_SIZE,
+                                                   self._RESPONSE_TIME_MAX,
+                                                   response_time)
```

Comment:
Just confirming...does this need something like 253: location = response.headers['location'] from the else branch above on the last chunk? (We had some issues related to whether or not the last chunk has the location header, maybe it's on the first chunk..)